### DISCUSSION TITLE
transfer domain from vercel to cloudflare

### DISCUSSION THREAD

**Discussion by domluna** [vercel/community#5456] [Account Issue]
### Question

I'm trying to transfer my domain to cloudflare from vercel. As far as I know the auth code worked. I have the domain on cloudflare but I also have it on vercel as well - it does not say it's been moved to a third party.

in the cloudflare page i get

Domain Registration
Registrar:
Tucows Domains

Transfer to Cloudflare

transfer conditions not met

it's been registered on vercel for > 60 days.

in order to get it to work I had to update the DNS records on Vercel.

how can I finalize the transfer?

**Comment by vadimsdev** [comment]
Do you still have the same issue or has your domain been transferred at the end? Asking because it seems I have the same issue right now...

**Reply by vadimsdev** [reply]
My domain is vadims.dev

![image](https://github.com/vercel/community/assets/20427712/30929692-1a73-4b70-83e6-129a08e0005f)

[a comment by an automated account is not shown]
